docs/release-notes/3.3.0.md

```diff
@@ -0,0 +1,2 @@
+## Bug Fixes
+- We've fixed a bug where APIRule v2alpha1 and v2 were put in an Error state when the target workload was in the mesh with the sidecar as an init container. See [#2100](https://github.com/kyma-project/api-gateway/issues/2100).
```

Comment:
```suggestion
- We've fixed a bug where APIRule v2alpha1 and v2 were put in an Error state when the target workload was in the mesh with the Istio proxy running in native sidecar mode. See [#2100](https://github.com/kyma-project/api-gateway/issues/2100).
```

The `sidecar as init container` is a sort of tautology, as k8s docs state the following:

> Kubernetes implements sidecar containers as a special case of [init containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/);

So, sidecar is an init-container by the (new) definition.